src/include/svrfunc.h

```diff
@@ -54,6 +54,9 @@ extern "C" {
 #include "pbs_sched.h"
 #include "pbs_entlim.h"
 
+#define SJ_IDX_2_TBLIDX(pj, sjidx) ((int)((sjidx - pj->ji_ajtrk->tkm_start) / pj->ji_ajtrk->tkm_step))
```

Comment:
I would suggest you add a line or two of the comments explaining what this macro is.